Assets/PatchKit Patcher/Scripts/Api/Models/Main/Versions.cs

```diff
@@ -0,0 +1,37 @@
+using Newtonsoft.Json;
+
+namespace PatchKit.Api.Models.Main
+{
+    public struct Versions
```

Comment:
I would suggest to rename it into ChangelogEntry as it might introduce some naming mismatch.